test/net/sourceforge/kolmafia/MonsterDataTest.java

```diff
@@ -425,6 +429,56 @@ void itemDropsAreRenderedProperly(final String monsterName, final String itemDro
     }
   }
 
+  @Nested
+  class JustTheFacts {
+    @Test
+    void characterWithoutFactSkillDisplaysNothing() {
+      var monster = MonsterDatabase.findMonster("fluffy bunny");
+
+      var cleanups = Player.withoutSkill(SkillPool.JUST_THE_FACTS);
+
+      try (cleanups) {
+        var builder = new StringBuilder();
+        monster.appendFact(builder, true);
+        assertThat(builder.toString(), not(containsString("Just the Facts: ")));
+      }
+    }
+
+    @Test
+    void factsAreRenderedIfUnrestricted() {
+      var monster = MonsterDatabase.findMonster("fluffy bunny");
+
+      var cleanups =
+          new Cleanups(
+              withSkill(SkillPool.JUST_THE_FACTS),
+              withNotAllowedInStandard(RestrictedItemType.SKILLS, "just the facts"),
```

Comment:
Since the canonical name of the skill is "Just the Facts" perhaps the same capitalization should be used in the calls to withNotAllowedInStandard?  Not a blocker.

Reply:
the key is lowercased when passed to the standard restricting function so I think this is fine

Reply:
It's more of a cognition thing. If the canonical name is Just the Facts then why not use it rather than assume the reader knows it will be lowercased before use?  And since I am lazy I will copy the string, thinking it is canonical and use it in a context where it must be canonical and so fuzzy matching or a case sensitive compare will mean my code is wrong.  But not a blocker so...